applications/box.md

```diff
@@ -0,0 +1,213 @@
+  
+  * An indication of how your project relates to / integrates into Crust: We will use crust as a pinning service to allow extra level of resiliance for those who want to have their files pinned out of their box
+  
+  * An indication of why your team is interested in creating this project:
+  
+  It was two years ago when cryptocurrencies and blockchain were the news page highlight, yet again. Everyone was excited about the potential of blockchain for decentralizing traditional central banks, however, the news didn’t have many headlines about the work that was being done to decentralize the whole web.
+
+At that time my friend, an AI geek, and I, a full-stack developer,  discussed this and started to search on the internet. We came across several promising projects aimed to decentralize the web and finally, after analysing many of them, IPFS and IPLD, both open-source, caught our eyes.
+
+They seemed promising to us, since they had a mature structure and a big community of open-source developers contributing to them.
+
+So we thought, this was great, why were people not using them instead of central storage servers? And the answer was not hard.
+
+As we spoke to our developer and non-developer friends, it became apparent that even many developers were not aware of these projects and many non-developers did not have the technical knowledge to set them up on their computers, even though it was just a few lines of command.
+
+Besides, there were not many applications that could connect to them or their UX were not as appealing as those developed by big-tech.
+
+We thought a framework that combines easy-to-use backend APIs, easy enough for an average front-end developer, and a pre-setup hardware, plug and play for a non-technical person, can fill this gap and bring decentralized web to everyone’s home.
+
+### Project Details 
+We expect the teams to already have a solid idea about your project's expected final state. Therefore, we ask the teams to submit (with urls or IPFS file CIDs included) at following :
+
+* Mockups/designs of any UI components: We are working on our first DApp that works with IPFS network and box. It is a replacement for "Google Photos" that let the users to store their photos on a "box" and on IPFS. Github project is here: [https://github.com/functionland/photos](https://github.com/functionland/photos)
+
+Below is a GIF demo from "Photos" application, which is a react native app, as the first demo of how an application interacts with "box"
+
+![https://dev.to/fx/google-photos-open-source-alternative-with-react-native-80c](https://raw.githubusercontent.com/functionland/blog/main/photos-intro-dev-to/photos.gif)
+
+* Data models / API specifications of the core functionality
```

Comment:
This picture shows you are using Filecoin to store user data. Is there any difference in the usage between Filecoin and Crust?

Reply:
Sorry, that was the initial photo and I forgot to add crust to the image. The use cases for Crust and Filecoin are different and box connects the user to both networks. However, per our research Filecoin primary usecase is a blockchain-style guarantee for high longevity that file is stored forever on the blockchain, on the other hand, Crust is about high availability to ensure user that they can have the files at anytime for any duration they want. The box as a package can be used on any hardware(e.g. laptop) and connects the user and their box applications to crust network for file pinning, which gives the user a way to try the package. We give both otpions as the use cases are different. In addition, Crust plans for "adding file sharing links" and "privacy storage", might enable us to partner with crust to provide box-as-a-service to users to enable them try the features without owning any hardware.

```suggestion
57 - ![https://dev.to/fx/google-photos-open-source-alternative-with-react-native-80c](https://raw.githubusercontent.com/functionland/functionland.github.io/main/static/images/box-diagram.jpg)
57 + ![https://dev.to/fx/google-photos-open-source-alternative-with-react-native-80c](https://github.com/functionland/functionland.github.io/blob/2883b03125087d59693e31628d59367d56206d56/static/images/box-diagram-crust.jpg)
```